source/MaterialXFormat/Util.cpp

```diff
@@ -98,25 +98,60 @@ void loadLibrary(const FilePath& file, DocumentPtr doc, const FileSearchPath* se
     doc->importLibrary(libDoc, &copyOptions);
 }
 
-StringVec loadLibraries(const FilePathVec& libraryNames,
+StringSet loadLibraries(const FilePathVec& libraryFolders,
                         const FileSearchPath& searchPath,
                         DocumentPtr doc,
                         const StringSet* excludeFiles,
                         XmlReadOptions* readOptions)
 {
-    StringVec loadedLibraries;
-    for (const std::string& libraryName : libraryNames)
+    // Include pathes specified by environment variable last
+    FileSearchPath librarySearchPath = searchPath;
+    librarySearchPath.append(getEnvironmentPath());
```

Comment:
Question posed for discussion: Should this environment variable be different dedicated just for libraries or should the current one be reused?

Reply:
I think it's good to include the environment path in the search. But maybe this could be done externally, appending to `searchPath` before the call. Then the caller can choose what directories to include and not include.

Reply:
Whether the environment path is appended internally or externally to this method, I like the idea of using the standard MATERIALX_SEARCH_PATH variable, rather than creating a new one for standard libraries.  This mirrors the approach we're currently using in MaterialXView, where the "--path" input is respected by standard libraries in addition to explicit XInclude references and image dependencies.

Reply:
Sounds good, and if we think there is never a reason to exclude searching the MATERIALX_SEARCH_PATH when loading libraries then we could do it internally, as done here.

Reply:
Checking with Xinclude logic it is indeed always added and is appended as the last search path so it should all be consistent.